game-core/src/main/java/games/strategy/engine/data/gameparser/GameParser.java

```diff
@@ -895,16 +895,16 @@ private Attachable findAttachment(
       }
       // decapitalize the property name for backwards compatibility
       final String name = LegacyPropertyMapper.mapLegacyOptionName(decapitalize(option.getName()));
-      if (LegacyPropertyMapper.ignoreOptionName(name)) {
```

Comment:
Would you mind explaining why this if block is moved down @trevan ?

If we are going to ignore an option, why should we execute `option.getValue()` or `option.getCount()`, wouldn't those just be dead stores?

Reply:
I moved it because I needed access to `option.getValue()` and I thought it would make more sense to use the local variable `value` that was going to be created vs passing in `option.getValue()`.  But I can move it back up and pass in `option.getValue()`.

Reply:
The local variable IMO is legacy, it's an unnecessary intermediate variable. This class in general followed a C++ pattern (if not older actually, maybe a C pattern or even Pascal) to declar all local variables at the beginning of a method. IMO all usages of 'value' probably should just get inlined.

I did not spot you were using the `value` variable. That answers my question.